app/api/csv/typeParsers/geolocation.js

```diff
@@ -0,0 +1,10 @@
+const geolocation = (entityToImport, templateProperty) => {
+  const [lat, lon] = entityToImport[templateProperty.name].split('|');
+  if (lat && lon) {
```

Comment:
Well, there is actually a place on earth with lat: 0 and lon: 0.  The chances are slim, but if anyone is using the geographic 0, 0... this will return null.

Reply:
Actually, any lat OR lon at 0 will return null, so chances are much less slim... this needs correcting.

Reply:
@RafaPolit this was working properly because '0' evaluates to true, anyway good catch, i wrote a a test anyways to demonstrate.